manifests/init.pp

```diff
@@ -15,6 +15,7 @@
   $ext_sshd_AuthorizedKeysCommand           = undef,
   $ext_sshd_AuthorizedKeysCommandUser       = 'nobody',
   $ext_sshd_ChallengeResponseAuthentication = 'no',
+  $ext_ssh_ForwardAgent                     = 'no',
```

Comment:
please add one extra test, where you modify the default value for $ext_ssh_ForwardAgent